File: Examen/Fortran/CelsiusAFahrenheit.f
      PROGRAM CelsiusAFahrenheit
      REAL C, F

      PRINT *, 'Ingrese la temperatura en grados Celsius:'
      READ *, C

      F = (C * 9.0 / 5.0) + 32.0
      PRINT *, 'La temperatura en Fahrenheit es: ', F

      PRINT *, 'Presione ENTER para salir...'
      READ(*,*)
      END

File: Examen/Java/CelsiusAFahrenheit.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Other/File.java to edit this template
 */

/**
 *
 * @author gyael
 */
import java.util.Scanner;

public class CelsiusAFahrenheit {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Ingrese la temperatura en grados Celsius: ");
        double celsius = sc.nextDouble();

        double fahrenheit = (celsius * 9 / 5) + 32;
        System.out.printf("La temperatura en Fahrenheit es: %.2f\n", fahrenheit);

        sc.close();
    }
}

